### PR TITLE
PYTHON-3383 Require and bundle libmongocrypt 1.5.2

### PR DESCRIPTION
**Pull request by ShaneHarvey**

### CODE REVIEW

**Review by blink1073 — APPROVED** [review]
👍🏼

File: bindings/python/release.sh [hunk 1]
```diff
@@ -66,7 +62,20 @@ elif [ "Darwin" = "$(uname -s)" ]; then
...
+    tar xzf libmongocrypt.tar.gz -C ./libmongocrypt
+    NOCRYPTO_SO=libmongocrypt/nocrypto/lib/libmongocrypt.dylib
+    chmod +x ${NOCRYPTO_SO}
+    cp ${NOCRYPTO_SO} pymongocrypt/
+    rm -rf ./libmongocrypt libmongocrypt.tar.gz
+
+    $PYTHON setup.py bdist_wheel
+
+    # Build the universal mac wheel
+    PYTHON="/Library/Frameworks/Python.framework/Versions/3.10/bin/python3"
```
Comment (ShaneHarvey) [inline bindings/python/release.sh:78]: @blink1073 I know this is going to conflict with your work but I want to get this in now to make the 1.3.1 release easier.

Comment (blink1073) [reply bindings/python/release.sh:78]: yep, sounds good

Comment (ShaneHarvey) [reply bindings/python/release.sh:78]: Here's what gets built on mac locally after these changes:
```
pymongocrypt-1.3.1-py2.py3-none-macosx_10_14_x86_64.whl
pymongocrypt-1.3.1-py2.py3-none-macosx_11_0_universal2.whl
pymongocrypt-1.3.1-py2.py3-none-manylinux2010_x86_64.whl
pymongocrypt-1.3.1-py2.py3-none-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
pymongocrypt-1.3.1.tar.gz
```

File: bindings/python/CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+Changes in Version 1.3.1
```
Comment (behackett) [inline bindings/python/CHANGELOG.rst:4]: We should say this is a recommended upgrade for all users of 1.3.0 with the release notes being discussed elsewhere.

Comment (ShaneHarvey) [reply bindings/python/CHANGELOG.rst:4]: Done.

**Pull request merged**